modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -2015,6 +2017,61 @@ public void testBuildTemplateServiceBuilder71() throws Exception {
 			packageName, "");
 	}
 
+	@Test
+	public void testBuildTemplateServiceBuilderCheckExports() throws Exception {
+		String name = "guestbook";
+		String packageName = "com.liferay.docs.guestbook";
+
+		String apiProjectName = name + "-api";
```

Comment:
The `apiProjectName` variable is used on `L2051` so it should be added there to improve readability.  I'll refer to this as the `as used` rule.  Can we update `L2022 - L2059` to this:

```
String name = "guestbook";
String packageName = "com.liferay.docs.guestbook";

File gradleProjectDir = _buildTemplateWithGradle(
	"service-builder", name, "--package-name", packageName,
	"--liferayVersion", "7.1");

File gradleServiceXml = new File(
	new File(gradleProjectDir, name + "-service"), "service.xml");

Consumer<Document> consumer = document -> {
	Element documentElement = document.getDocumentElement();

	documentElement.setAttribute("package-path", "com.liferay.test");
};

_editXml(gradleServiceXml, consumer);

File mavenProjectDir = _buildTemplateWithMaven(
	"service-builder", name, "com.test", "-Dpackage=" + packageName,
	"-DliferayVersion=7.1");

File mavenServiceXml = new File(
	new File(mavenProjectDir, name + "-service"), "service.xml");

_editXml(mavenServiceXml, consumer);

_testContains(
	gradleProjectDir, name + "-api/bnd.bnd", "Export-Package:\\",
	packageName + ".exception,\\", packageName + ".model,\\",
	packageName + ".service,\\", packageName + ".service.persistence");

Optional<String> stdOutput = _executeGradle(
	gradleProjectDir, false, true,
	name + "-service" + _GRADLE_TASK_PATH_BUILD);

Assert.assertTrue(stdOutput.isPresent());
```